### ISSUE TITLE
Travis build fails with Redmine 2.6.3 and Redmine 3.0.1 (don't use these versions!)

### ISSUE THREAD

**Issue by n-rodriguez** [jbox-web/redmine_git_hosting#387]
```
 1) Repository::Xitolite common_tests : long tests Test uniqueness when blank identifier is already taken by a repository should be invalid

Failure/Error: it { expect(build_git_repository(project: @project_child, identifier: '')).to be_invalid }

expected `#<Repository::Xitolite id: nil, project_id: 157, url: "repositories/redmine/project-parent/project-child....", login: "", password: "", root_url: "repositories/redmine/project-parent/project-child....", type: "Repository::Xitolite", path_encoding: nil, log_encoding: nil, extra_info: nil, identifier: "", is_default: false, created_on: nil>.invalid?` 
to return true, got false

# ./plugins/redmine_git_hosting/spec/models/repository_xitolite_spec.rb:252:in `block (5 levels) in <top (required)>'
```

**Comment by n-rodriguez** [comment]
Root cause : http://www.redmine.org/issues/19260

**Comment by n-rodriguez** [comment]
It allows this kind of things 
![configuration - test - redmine](https://cloud.githubusercontent.com/assets/3433835/6678749/bd6ec1a8-cc3e-11e4-8793-f55849e0edcb.png)

**Comment by n-rodriguez** [comment]
[Redmine issue](http://www.redmine.org/issues/19400#change-62305)

**Comment by perltwiesekt** [comment]
Heja, can you confirm this is fixed in Redmine 2.6.4?

**Comment by n-rodriguez** [comment]
Unfortunately not : https://travis-ci.org/jbox-web/redmine_git_hosting/jobs/58032852

**Comment by shaneonabike** [comment]
Is this fixed in 3.0.1 because I seem to be having issues installing it?

**Comment by n-rodriguez** [comment]
> Is this fixed in 3.0.1 because I seem to be having issues installing it?

Niope. You **must** not use this version.

**Comment by shaneonabike** [comment]
Ah that's tragic! Guess I'll go back to reinstalling it again..
